aten/src/ATen/CMakeLists.txt

```diff
@@ -415,6 +415,12 @@ if(NOT MSVC AND NOT EMSCRIPTEN AND NOT INTERN_BUILD_MOBILE)
 endif()
 
 if(USE_CUDA AND NOT USE_ROCM)
+  # add_subdirectory("${CMAKE_CURRENT_SOURCE_DIR}/../../../third_party/cutlass" ${CMAKE_BINARY_DIR}/cutlass)
+  list(APPEND ATen_THIRD_PARTY_INCLUDE ${TORCH_ROOT}/third_party/cutlass/include)
+  list(APPEND ATen_CUDA_INCLUDE ${CMAKE_CURRENT_SOURCE_DIR}/../../../third_party/cutlass/include)
+  message("!!!!!!!!!!!!!!!!!THIRD PARTY INCLUDES BRO: ${ATen_THIRD_PARTY_INCLUDE}!!!!!!!!!!!!!!!!!")
```

Comment:
whoops.. cmake was getting frustrating.. I think that line is actually what is needed so that when building the cu kernel cutlasss includes can be found